src/Compilers/Core/Portable/InternalUtilities/ThreeState.cs

```diff
@@ -31,5 +31,10 @@ public static bool Value(this ThreeState value)
             Debug.Assert(value != ThreeState.Unknown);
             return value == ThreeState.True;
         }
+
+        public static bool IsTrue(this ThreeState value)
+        {
+            return value == ThreeState.True;
+        }
```

Comment:
no, i was going to use it but ended up not needing it

---
In reply to: [118737003](https://github.com/dotnet/roslyn/pull/19795#discussion_r118737003) [](ancestors = 118737003)